target/linux/ramips/dts/mt7621.dtsi

```diff
@@ -366,7 +386,7 @@
 		       0x1e1d0700 0x0100>;
 		reg-names = "mac", "ippc";
 
-		clocks = <&sysclock>;
+		clocks = <&sysc MT7621_CLK_50M>;
 		clock-names = "sys_ck";
```

Comment:
I'm not sure if USB xhci needs to use `MT7621_CLK_XTAL`(40 MHz). The original frequency in sysclock node is 50 MHz.

Reply:
@DragonBluep Previous dtsi when the clock driver was mainlined had 50 MHz for the USB so that is the reason for that frequency. I have USB working in one of my boards with 50 MHz frequency without issues but it is true that the uboot reference is using XTAL clock for that.

Reply:
@DragonBluep BTW is correct on mainline. It uses XTAL: https://elixir.bootlin.com/linux/v6.4-rc7/source/arch/mips/boot/dts/ralink/mt7621.dtsi

Reply:
Finally, I chose to use XTAL. It looks like everything is normal. At a glance, it seems that the USB driver does not care about these clocks.